### ISSUE TITLE
Improve error detection scenarios on Stage and Migrate

### ISSUE THREAD

**Issue by jwmatthews** [fusor/mig-ui#243]
mig-controller should now have error detection via
 https://github.com/fusor/mig-controller/pull/136

Issue: https://github.com/fusor/mig-controller/issues/85

**Comment by jwmatthews** [comment]
Ran a Migrate today of a 'Move' that failed.

As it failed the UI displayed a flash message of 'Migration Successful', yet screenshot shows it 'Failed'

<img width="1653" alt="Screen Shot 2019-07-06 at 11 26 27 AM" src="https://user-images.githubusercontent.com/206875/60758095-280e6200-9fe1-11e9-8c11-cb02bcd06173.png">


From destination (ocp4 side)
```
$ oc get migmigrations 479fe230-a002-11e9-9116-2b21d47c436a -o yaml
apiVersion: migration.openshift.io/v1alpha1
kind: MigMigration
metadata:
  annotations:
    touch: e13cb5be-3ffc-4163-b3cc-7157404dfe2b
  creationTimestamp: "2019-07-06T15:25:26Z"
  generation: 4
  name: 479fe230-a002-11e9-9116-2b21d47c436a
  namespace: mig
  resourceVersion: "60918"
  selfLink: /apis/migration.openshift.io/v1alpha1/namespaces/mig/migmigrations/479fe230-a002-11e9-9116-2b21d47c436a
  uid: 47b0a634-a002-11e9-93ff-0af4467123f4
spec:
  migPlanRef:
    name: mysql
    namespace: mig
  quiescePods: true
  stage: false
status:
  conditions:
  - category: Advisory
    lastTransitionTime: "2019-07-06T15:26:20Z"
    message: The migration has completed successfully.
    reason: BackupFailed
    status: "True"
    type: Succeeded
  - category: Critical
    lastTransitionTime: "2019-07-06T15:26:20Z"
    message: 'The migration has failed.  See: Errors.'
    reason: BackupFailed
    status: "True"
    type: Failed
  errors:
  - 'Backup: mig/479fe230-a002-11e9-9116-2b21d47c436a-lk7th failed.'
  phase: BackupFailed
  startTimestamp: "2019-07-06T15:25:26Z"

```


From source (ocp3) side
```

$ oc get backups 479fe230-a002-11e9-9116-2b21d47c436a-lk7th -o yaml
apiVersion: velero.io/v1
kind: Backup
metadata:
  annotations:
    openshift.io/migrate-copy-phase: final
    openshift.io/migrate-quiesce-pods: "true"
    openshift.io/migration-registry: 172.30.229.85:5000
    openshift.io/migration-registry-dir: /mysql-registry-86203e72-9ff6-11e9-93ff-0af4467123f4
  creationTimestamp: "2019-07-06T15:25:55Z"
  generateName: 479fe230-a002-11e9-9116-2b21d47c436a-
  generation: 1
  labels:
    migmigration: 47b0a634-a002-11e9-93ff-0af4467123f4
    velero.io/storage-location: jmatthewsvelero-tfvd7
  name: 479fe230-a002-11e9-9116-2b21d47c436a-lk7th
  namespace: mig
  resourceVersion: "133233"
  selfLink: /apis/velero.io/v1/namespaces/mig/backups/479fe230-a002-11e9-9116-2b21d47c436a-lk7th
  uid: 58df43bf-a002-11e9-acb7-0a9e3bf90304
spec:
  excludedNamespaces: []
  excludedResources: []
  hooks:
    resources: []
  includeClusterResources: null
  includedNamespaces:
  - mysql-persistent
  includedResources: []
  labelSelector: null
  storageLocation: jmatthewsvelero-tfvd7
  ttl: 720h0m0s
  volumeSnapshotLocations:
  - jmatthewsvelero-mwhdv
status:
  completionTimestamp: "2019-07-06T15:25:55Z"
  errors: 0
  expiration: "2019-08-05T15:25:55Z"
  phase: Failed
  startTimestamp: "2019-07-06T15:25:55Z"
  validationErrors: null
  version: 1
  volumeSnapshotsAttempted: 0
  volumeSnapshotsCompleted: 0
  warnings: 0

```

Looking at Velero controller logs, I think the issue is related to Velero unable to interact with the S3 bucket.

**Comment by eriknelson** [comment]
Should be fixed with #434
